catroid/src/androidTest/java/org/catrobat/catroid/test/BricksHelpUrlTest.java

```diff
@@ -428,6 +428,8 @@
 				"https://wiki.catrobat.org/bin/view/Documentation/BrickDocumentation/StartListeningBrick");
 		brickToHelpUrlMapping.put("org.catrobat.catroid.content.bricks.SetListeningLanguageBrick",
 				"https://wiki.catrobat.org/bin/view/Documentation/BrickDocumentation/SetListeningLanguageBrick");
+		brickToHelpUrlMapping.put("org.catrobat.catroid.content.bricks.PauseForBeatsBrick",
+				"https://wiki.catrobat.org/bin/view/Documentation/Brick%20Documentation/PauseForBeatsBrick");
```

Comment:
```suggestion
				"https://wiki.catrobat.org/bin/view/Documentation/BrickDocumentation/PauseForBeatsBrick");
```
Sorry, changed since yesterday.